projects/packages/search/src/class-rest-controller.php

```diff
@@ -151,6 +153,15 @@ protected function register_jetpack_only_rest_routes() {
 				'permission_callback' => 'is_user_logged_in',
 			)
 		);
+		register_rest_route(
+			static::$namespace,
+			'/search/local-stats',
```

Comment:
(Future PR) Can we add a comment describing the difference between `/search/local-stats` and `/search/stats`?

Reply:
Sure thing! I'll explain here just for future reference:
- `/search/stats` is forwarded to WPCOM and has post type breakdown data from ES index.
- `/search/local-stats` has post type breakdown data generated from the local site database.